Docs/source/usage/parameters.rst

```diff
@@ -2522,6 +2522,57 @@ Reduced Diagnostics
            for a uniform grid simulation.
 
 
+    * ``RawBFieldReduction``
+        This type is ONLY for the B-field at their respective staggering on the Yee-grid (or the type of grid used in the
+        simulation) and executes the ``reduced_function`` which is a user-defined analytic function as given below.
+
+        * ``<reduced_diags_name>.reduced_function(x,y,z)`` (`string`)
+              An analytic function used to select the region over which the B-fields will be reduced using
+              the ``reduction_type`` described below.
+
+        * ``<reduced_diags_name>.reduction_type`` (`string`)
+            The type of reduction to be performed. It must be either ``Maximum``, ``Minimum`` or
+            ``Integral``.
+            ``Integral`` computes the spatial surface or volume integral, depending on the choice
+            of the ``integration_type``, of the function defined in the parser by summing its value on
+            all grid points and multiplying the result by the area or volume of a cell.
+            Please be also aware that measuring maximum quantities might be very noisy in PIC
+            simulations.
+
+        The output columns correspond to the timestep counter, physical time, and reduced values of Bx, By, Bz components.
+
+        * ``<reduced_diags_name>.integration_type`` (`string`)
+           The type of integration to be performed. It must be either ``surface`` or ``volume``.
+           Note that the surface integral provides the integrated normal as well as the
+           traverse fields over the surface. For the surface integral, the user-defined parser
+           for the plane should be less than one cell size in thickness in the direction of the surface normal.
+           Also, if the surface to be defined has an edge that overlaps with the domain boundary but the edge parallel
+           to it does not overlap with the domain boundary, e.g. a half cross-section of a plane, then exclude the
+           edge that overlaps with the domain boundary while defining the surface.
```

Comment:
is this valud for the B-field defined at the face-center. The normal component of B-field is on the face-center for yee

Reply:
Yes.